LanguageFeatures/Private-named-parameters/static_semantics_A01_t01.dart

```diff
@@ -7,47 +7,47 @@
 /// - If `p` has no corresponding public name `n`, then compile-time error.
 ///
 /// @description Check that it is a compile-time error if a named initializing
-/// formal parameter with private name has no corresponding public name.
+/// formal parameter has a private name that has no corresponding public name.
 /// @author sgrekhov22@gmail.com
 
 // SharedOptions=--enable-experiment=private-named-parameters
 
 class C {
-  String __p;
-  C({this._p = ""});
-//        ^^
+  String _p;
+  C({this.__p = ""});
```

Comment:
It is true that `__p` has no corresponding public name, but we will also have an error because there is no instance variable named `__p`, so line 16 should be changed back to `__p`. With that we should only get the error caused by the lack of a corresponding public name for the parameter.

Reply:
Yes, indeed. For `__p` the public name is `_p` which is not public and therefore not the public name. But the instance variable name still should be `__p`. Sorry I was confused. Fixed.

Reply:
Thanks!